pkg/controller/support/credentials/credentials.go

```diff
@@ -136,7 +147,9 @@ func Verify(ctx context.Context, configDir string, insecureRegistries []string)
 		auth.ServerAddress = server
 
 		if _, _, err = svc.Auth(ctx, &auth, "DominoDataLab_Hephaestus/1.0"); err != nil {
-			errs = append(errs, fmt.Errorf("%q client credentials are invalid: %w", server, err))
+			//nolint:lll
+			detailedErr := fmt.Errorf("%q client credentials are invalid. Make sure the provided credentials are correct: %s. Underlying error: %w", server, strings.Join(helpMessage, ","), err)
```

Comment:
Not 100% sure how the formatting looks here, so apologies if it's weird. Just looking for a bit more readability for users.
```suggestion
			detailedErr := fmt.Errorf("unable to authenticate to registry %q\nMake sure the following sources of credentials are correct: %s\nUnderlying error: %w", server, strings.Join(helpMessage, ", "), err)
```

or maybe

```golang
detailedErr := fmt.Errorf("unable to authenticate to registry %q\nMake sure the following sources of credentials are correct: %s", server, strings.Join(helpMessage, ", "))
errs = append(errs, detailedErr, err)
```